packages/ringcentral-widgets/components/ConferenceParticipantPanel/index.js

```diff
@@ -131,12 +139,22 @@ ParticipantsContainer.propTypes = {
   participants: PropTypes.arrayOf(PropTypes.object).isRequired,
   onBackButtonClick: PropTypes.func,
   formatPhone: PropTypes.func,
+  removeParticipantClickRemoveTrack: PropTypes.func,
```

Comment:
remove the redundant props: `removeParticipantClickRemoveTrack `, `removeParticipantClickCancelTrack ` and `participantListClickHangupTrack `